app-editors/atom/files/gyp-unbundle.py

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/env python
```

Comment:
Where does this file come from? Please specify the version of Python i.e. `python2` or `python3`.